src/utilities/TID300/index.js

```diff
@@ -0,0 +1,123 @@
+// Should specify the Mean Modality Pixel Value measured in whatever units the image is in
+// Should specify the Standard Deviation Modality Pixel Value measured in whatever units the image is in
+//
+// 4. Cornerstone Ellipse:
+//
+// Hierarchy
+// TID 1500 MeasurementReport
+// -TID 1410 Planar ROI Measurements
+// --TID 1501 Measurement Group
+// ---Measurement Group (DCM 125007)
+// ----TID 300 Measurement
+// ------SCOORD. Graphic Type: ELLIPSE
+//        (ftp://dicom.nema.org/MEDICAL/dicom/current/output/chtml/part03/sect_C.10.5.html)
+//
+// If Graphic Type (0070,0023) is ELLIPSE, then exactly four points shall be present; the first two points are to be interpreted as the endpoints of the major axis and the second two points as the endpoints of the minor axis of an ellipse, some form of implementation dependent representation of which is to be drawn.
+//
+// TID 1401 Area Measurement: http://dicom.nema.org/medical/dicom/current/output/html/part16.html#sect_TID_1401
+// Should be a sibling of the SCOORD
+// Should specify the Mean Modality Pixel Value measured in whatever units the image is in
+// Should specify the Standard Deviation Modality Pixel Value measured in whatever units the image is in
+//
+//
+// 5. Cornerstone Rectangle ROI
+//
+// Hierarchy
+// TID 1500 MeasurementReport
+// --TID 1501 Measurement Group
+// ---Measurement Group (DCM 125007)
+// ----TID 300 Measurement
+// ------SCOORD. Graphic Type: POLYLINE
+// ------ TID 4019 Algorithm Identification:
+//                 AlgorithmName: 'Rectangle'
+//                 AlgorithmVersion: 'Cornerstone'
+//
+//                 http://dicom.nema.org/medical/dicom/current/output/html/part16.html#sect_TID_4019
+//
+// OR
+// Note: This should be the same as a Freehand ROI, more or less. We add a TID 4019: Algorithm Identification flag to specify that this was created (and should be rehydrated) into a Rectangle ROI.
+// TODO: Should we use a Derivation instead? http://dicom.nema.org/medical/dicom/current/output/html/part16.html#DCM_121401
+// Should specify the Area measured in mmˆ2, including the units in UCUM
+// Should specify the Mean Modality Pixel Value measured in whatever units the image is in
+// Should specify the Standard Deviation Modality Pixel Value measured in whatever units the image is in
+//
+// TODO: Should we specify the
+//
+//
+// 6. Cornerstone Simple Angle tool
+//
+// Hierarchy
+// TID 1500 MeasurementReport
+// --TID 1501 Measurement Group
+// ---Measurement Group (DCM 125007)
+// ----TID 300 Measurement
+// ------SCOORD. Graphic Type: POLYLINE
+//        (ftp://dicom.nema.org/MEDICAL/dicom/current/output/chtml/part03/sect_C.10.5.html)
+// ----TID 300 Measurement
+// ------SCOORD. Graphic Type: POLYLINE
+//        (ftp://dicom.nema.org/MEDICAL/dicom/current/output/chtml/part03/sect_C.10.5.html)
+//
+// ------ TID 4019 Algorithm Identification:
+//                 AlgorithmName: 'Angle'
+//                 AlgorithmVersion: 'Cornerstone'
```

Comment:
Thinking about this, I would imagine that the concept corresponding to the angle measurement would be sufficient. Can you think of an example where interpretation of the annotation would be ambiguous for a specific angle measurement type defined by the quantity concept?

Reply:
In any case, I would not use Algorithm identification to communicate the semantics of the annotation.